src/Microsoft.AspNet.Mvc.Core/FromServicesAttribute.cs

```diff
@@ -7,25 +7,9 @@
 namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
-    /// Specifies that a parameter or property should be bound using the request services.
+    /// Specifies that an action parameter should be bound using the request services.
     /// </summary>
     /// <example>
-    /// In this example, the LocationService property on the VehicleWithDealerViewModel class 
-    /// will be bound to the value resolved for the ILocationService service from the request services.
-    ///
-    /// <code> 
-    /// public class VehicleWithDealerViewModel 
-    /// {
-    ///     [FromServices]
-    ///     public ILocationService LocationService { get; set; }
-    ///
-    ///     public void Update()
-    ///     {
-    ///         LocationService.Update();
-    ///     }
-    /// }
-    /// </code> 
```

Comment:
Property injection doesn't work any more. The other example talks about parameter injection and I've left that alone.